MAVProxy/modules/mavproxy_link.py

```diff
@@ -853,12 +846,8 @@ def master_callback(self, m, master):
                 if mtype_instance not in self.status.msg_count:
                     self.status.msg_count[mtype_instance] = 0
                 self.status.msg_count[mtype_instance] += 1
-        
-        if m.get_srcComponent() == mavutil.mavlink.MAV_COMP_ID_GIMBAL and mtype == 'HEARTBEAT':
-            # silence gimbal heartbeat packets for now
-            return
 
-        if getattr(m, 'time_boot_ms', None) is not None and self.settings.target_system == m.get_srcSystem():
+        if getattr(m, 'time_boot_ms', None) is not None and self.is_primary_vehicle(m):
```

Comment:
This is a good change - components won't necessarily have the same timestamp as the parent vehicle.